### ISSUE TITLE
[JENKINS-62419] Indicate the language of the page in the footer

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/jenkins#11539]
<p>It would be great to indicate the language of the page being displayed. Even if some localizations are missing, showing a baseline language might be useful.</p>

<p>This locale field should be somehow determined from the response page so that plugins like Locale Plugin are handled correctly (see the filter here: <a href="https://github.com/jenkinsci/locale-plugin/blob/master/src/main/java/hudson/plugins/locale/LocaleFilter.java" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/locale-plugin/blob/master/src/main/java/hudson/plugins/locale/LocaleFilter.java</a>)</p>

---
<details><summary><i>Originally reported by <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/oleg_nenashev.png" title="oleg_nenashev's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=oleg_nenashev">oleg_nenashev</a>, imported from: <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-62419" target="_blank">Indicate the language of the page in the footer</a></i></summary>
<i><ul>
<li><b>status</b>: Closed
<li><b>priority</b>: Minor
<li><b>component(s)</b>: core
<li><b>label(s)</b>: accessibility, newbie-friendly, user-experience
<li><b>resolution</b>: Won't Fix
<li><b>resolved</b>: 2025-03-21T13:23:03+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 3
<li><b>imported</b>: 2025-11-24
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>It would be great to indicate the language of the page being displayed. Even if some localizations are missing, showing a baseline language might be useful.</p>

<p>This locale field should be somehow determined from the response page so that plugins like Locale Plugin are handled correctly (see the filter here: <a href="https://github.com/jenkinsci/locale-plugin/blob/master/src/main/java/hudson/plugins/locale/LocaleFilter.java" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/locale-plugin/blob/master/src/main/java/hudson/plugins/locale/LocaleFilter.java</a>)</p></pre>
</details>
</details>
<details><summary><i>4 attachments</i></summary>

- [jenkins-locale-1.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51316/jenkins-locale-1.png)
  > ![jenkins-locale-1.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51316/jenkins-locale-1.png)
- [jenkins-locale-2.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51315/jenkins-locale-2.png)
  > ![jenkins-locale-2.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51315/jenkins-locale-2.png)
- [jenkins-locale-3.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51314/jenkins-locale-3.png)
  > ![jenkins-locale-3.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51314/jenkins-locale-3.png)
- [jenkins-locale-4.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51317/jenkins-locale-4.png)
  > ![jenkins-locale-4.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51317/jenkins-locale-4.png)
</details>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/aytuncbeken.png" title="aytuncbeken's avatar" /> aytuncbeken:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-62419?focusedId=391363&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-391363">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Hi,</p>

<p>Implemented changes in Jenkins Core with this branch. <a href="https://github.com/aytuncbeken/jenkins/tree/JENKINS-62419" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/aytuncbeken/jenkins/tree/JENKINS-62419</a></p>

<p>While implementing, I noticed that, Request object returns the true Locale information when Locale Plugin is used. Therefore, implementation is getting Locale info from Request object. </p>

<p> <span class="image-wrap" style=""><a id="51314_thumb" href="https://issues.jenkins.io/secure/attachment/51314/51314_jenkins-locale-3.png" title="jenkins-locale-3.png" file-preview-type="image" file-preview-id="51314" file-preview-title="jenkins-locale-3.png"><img src="https://issues.jenkins.io/secure/thumbnail/51314/_thumb_51314.png" style="border: 0px solid black" role="presentation"/></a></span>  <span class="image-wrap" style=""><a id="51315_thumb" href="https://issues.jenkins.io/secure/attachment/51315/51315_jenkins-locale-2.png" title="jenkins-locale-2.png" file-preview-type="image" file-preview-id="51315" file-preview-title="jenkins-locale-2.png"><img src="https://issues.jenkins.io/secure/thumbnail/51315/_thumb_51315.png" style="border: 0px solid black" role="presentation"/></a></span>  <span class="image-wrap" style=""><a id="51316_thumb" href="https://issues.jenkins.io/secure/attachment/51316/51316_jenkins-locale-1.png" title="jenkins-locale-1.png" file-preview-type="image" file-preview-id="51316" file-preview-title="jenkins-locale-1.png"><img src="https://issues.jenkins.io/secure/thumbnail/51316/_thumb_51316.png" style="border: 0px solid black" role="presentation"/></a></span> </p>

</pre>
</details>

</ul>
</details>

<p>Hi,</p>

<p>Implemented changes in Jenkins Core with this branch. <a href="https://github.com/aytuncbeken/jenkins/tree/JENKINS-62419" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/aytuncbeken/jenkins/tree/JENKINS-62419</a></p>

<p>While implementing, I noticed that, Request object returns the true Locale information when Locale Plugin is used. Therefore, implementation is getting Locale info from Request object. </p>

<p> <span class="image-wrap" style=""><a id="51314_thumb" href="https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51314/jenkins-locale-3.png" title="jenkins-locale-3.png" file-preview-type="image" file-preview-id="51314" file-preview-title="jenkins-locale-3.png"><img src="https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51314/jenkins-locale-3.png" style="border: 0px solid black" role="presentation"/></a></span>  <span class="image-wrap" style=""><a id="51315_thumb" href="https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51315/jenkins-locale-2.png" title="jenkins-locale-2.png" file-preview-type="image" file-preview-id="51315" file-preview-title="jenkins-locale-2.png"><img src="https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51315/jenkins-locale-2.png" style="border: 0px solid black" role="presentation"/></a></span>  <span class="image-wrap" style=""><a id="51316_thumb" href="https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51316/jenkins-locale-1.png" title="jenkins-locale-1.png" file-preview-type="image" file-preview-id="51316" file-preview-title="jenkins-locale-1.png"><img src="https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51316/jenkins-locale-1.png" style="border: 0px solid black" role="presentation"/></a></span> </p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>sakshi__:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-62419?focusedId=453001&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-453001">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Hi, is this issue resolved? or any new pull request regarding this issue has been made that I can work on?</p>
</pre>
</details>

</ul>
</details>

<p>Hi, is this issue resolved? or any new pull request regarding this issue has been made that I can work on?</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/timja.png" title="timja's avatar" /> timja:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-62419?focusedId=453275&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-453275">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>This isn't something we want, see <a href="https://github.com/jenkinsci/jenkins/pull/10419" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jenkins/pull/10419</a></p>
</pre>
</details>

</ul>
</details>

<p>This isn't something we want, see <a href="https://github.com/jenkinsci/jenkins/pull/10419" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jenkins/pull/10419</a></p>

**Comment by jenkins-infra-bot** [comment]
<i>[Original `epic` from Jira: <a href="https://github.com/jenkinsci/jenkins/issues?q=is%3Aissue%20%22jira_issue_key%3DJENKINS-62268%22">JENKINS-62268</a>]</i>
